src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -247,8 +247,10 @@ trait WorkflowComponent {
     }
 
     def loadFailedInputResolutions(failureId: Long): ReadAction[Seq[SubmissionValidationValue]] = {
-      val failedInputResolutionsAttributeJoin = findInputResolutionsByFailureId(failureId) join submissionAttributeQuery on (_.id === _.ownerId)
-      unmarshalInputResolutions(failedInputResolutionsAttributeJoin)
+      val failedInputResolutionsAttributeJoin = findInputResolutionsByFailureId(failureId) joinLeft submissionAttributeQuery on (_.id === _.ownerId)
```

Comment:
To make sure I'm understanding this change: before we were doing an inner join of input resolutions on submission attributes, but since there were potentially no associated submission attributes with those failed input resolutions, those particular input resolutions were being thrown out in the join. Now, you're preserving the input resolutions even if they don't have a corresponding submission attribute by doing a left join. I'm fairly new to the subtleties of the various joins so I want to make sure I've got it understood, but if I've got that reasoning right then I'll give a :+1: